KK-DZ/src/app/modules/workers/pages/workers-page/workers-facade.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from '@angular/core';
+
+import { Employee } from 'src/app/core/models/employee';
+import { EmployeeFiltersState } from 'src/app/core/models/employee-filters-state';
+
+import { WorkersService } from '../../services/workers.service';
+import { WorkersClientService } from '../../clients/workers-client.service';
+import { EmployeeEntity } from 'src/app/core/models/employee-entity';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class WorkersFacadeService {
+  public list$ = this.workersService.list$;
+  public filters$ = this.workersService.filters$;
+
+  constructor(private workersService: WorkersService, private workersClient: WorkersClientService) { }
+
+  public setFilters(filtersState: EmployeeFiltersState): void {
+    this.workersService.setFilters(filtersState);
+  }
+
+  public add(employee: Employee): void {
+    this.workersClient.post(employee).subscribe(worker => {
+      this.workersService.add(worker);
```

Comment:
Po każdej akcji typu add/edit/delete zamiast wykonywac specjalne metody zmieniające stan tj. workersService.add lepiej przeladowac liste workers poprzez loadWorkers(), wtedy mamy pewność że utrzymana jest synchronizacja miedzy klientem a bazą danych ( jeśli coś zle sie zapisało na backendzie to po przeladowaniu workers od razu będziemy mieli prawdziwy stan, a obecnie nie bedzie tego widać na UI)